response_operations_ui/static/js/data-panel.js

```diff
@@ -0,0 +1,21 @@
+function initDataPanels() {
+    var acc = document.getElementsByClassName("data-panel-header");
+    for (var i = 0; i < acc.length; i++) {
+        // Introduce button and hide data panel content if js running
+        var panelTitle = acc[i].innerHTML;
+        acc[i].innerHTML = "<button class='data-panel-header__title'>" + panelTitle + "<\/button>";
```

Comment:
It would be preferable to have the javascript toggle the visibility settings of the button rather than render html directly.

Reply:
I agree, I'll have a quick look